apps/site/src/app/(home)/sections/Sponsors/Sponsors.tsx

```diff
@@ -3,31 +3,35 @@ import { getSponsors } from "./getSponsors";
 import styles from "./Sponsors.module.scss";
 import { client } from "@/lib/sanity/client";
 import imageUrlBuilder from "@sanity/image-url";
-import clip from "./clip.svg";
+import Container from "react-bootstrap/Container";
```

Comment:
Chore: rearrange import order to be in the order of library components, our custom components, assets, then styles